app/experimenter/nimbus-ui/public/index.html

```diff
@@ -1,5 +1,5 @@
 <!--
-  NOTICE: This HTML file is no longer used. To modify the HTML file used
-  for this React app please make changes to the following file:
-  app/experimenter/legacy-ui/templates/experiments/nimbus-ui.html
+  NOTICE: This HTML file is not used but is required. To modify the HTML file
+  used for this React app please make changes to the following file:
+  app/experimenter/nimbus-ui/templates/nimbus/index.html
```

Comment:
The path needed an update, but I actually mistakenly deleted this file thinking we didn't need it at all which is why I added a "but is required" note.